CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 6.8.0
+- Add static types to IIB
```

Comment:
Few more PRs can be added in the changelog, like https://github.com/release-engineering/iib/pull/455 and https://github.com/release-engineering/iib/pull/451
Also, static types change was already released no?

Reply:
I just merged https://github.com/release-engineering/iib/pull/461 as well

Reply:
- Remove misleading log from API response
- Unsupported retry package replaced by tenacity

Reply:
Made the changes